java/src/main/java/com/powsybl/dataframe/network/adders/PhaseTapChangerDataframeAdder.java

```diff
@@ -0,0 +1,76 @@
+package com.powsybl.dataframe.network.adders;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.dataframe.SeriesMetadata;
+import com.powsybl.dataframe.update.UpdatingDataframe;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.PhaseTapChanger;
+import com.powsybl.iidm.network.PhaseTapChangerAdder;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+
+import java.util.List;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ * @author Sylvain Leclerc <sylvain.leclerc@rte-france.com>
+ */
+public class PhaseTapChangerDataframeAdder implements NetworkElementAdder {
+
+    private static final List<SeriesMetadata> METADATA = List.of(
+            SeriesMetadata.stringIndex("id"),
+            SeriesMetadata.strings("regulation_mode"),
+            SeriesMetadata.doubles("target_deadband"),
+            SeriesMetadata.ints("low_tap"),
+            SeriesMetadata.ints("tap")
+    );
+
+    private static final List<SeriesMetadata> STEPS_METADATA = List.of(
```

Comment:
For phase tap changers, I think that we can provide a lot of default values and maybe allow only angles to be set.

Reply:
See discussion synthesis about default values in comment below.